### ISSUE TITLE
ActionController::UnknownFormat for JSON references route

### ISSUE THREAD

**Issue by aepyornis** [public-accountability/littlesis-rails#1267]
```
Started GET "/org/37687-Elliott_Management/references" 
INFO -- : Processing by EntitiesController#references as JSON
 INFO -- :   Parameters: {"id"=>"37687-Elliott_Management"}
 INFO -- : Completed 406 Not Acceptable in 5ms (ActiveRecord: 0.6ms | Allocations: 1593)
FATAL -- :   
ActionController::UnknownFormat (EntitiesController#references is missing a template for this request format and variant.

request.formats: ["application/json"]
request.variant: []):
  
actionpack (6.0.3.4) lib/action_controller/metal/implicit_render.rb:42:in `default_render'
actionpack (6.0.3.4) lib/action_controller/metal/basic_implicit_render.rb:6:in `block in send_action'
actionpack (6.0.3.4) lib/action_controller/metal/basic_implicit_render.rb:6:in `tap'
actionpack (6.0.3.4) lib/action_controller/metal/basic_implicit_render.rb:6:in `send_action'
actionpack (6.0.3.4) lib/abstract_controller/base.rb:195:in `process_action'
actionpack (6.0.3.4) lib/action_controller/metal/rendering.rb:30:in `process_action'
actionpack (6.0.3.4) lib/abstract_controller/callbacks.rb:42:in `block in process_action'
activesupport (6.0.3.4) lib/active_support/callbacks.rb:135:in `run_callbacks'

```

**Comment by robjlucas** [comment]
Seems to be already fixed (probably in 31567c968b70465c16ea17c328a8ab1d1dce4c0a). If I visit http://localhost:8080/org/37687-Elliott_Management I get:

```
I, [2021-01-18T11:44:27.331976 #7]  INFO -- : Started GET "/references/entity?entity_id=37687&page=1" for 172.18.0.1 at 2021-01-18 11:44:27 +0000
I, [2021-01-18T11:44:27.352977 #7]  INFO -- : Processing by ReferencesController#entity as JSON
I, [2021-01-18T11:44:27.353166 #7]  INFO -- :   Parameters: {"entity_id"=>"37687", "page"=>"1"}
D, [2021-01-18T11:44:27.358820 #7] DEBUG -- :   User Load (1.9ms)  SELECT `users`.* FROM `users` WHERE `users`.`id` = 1 ORDER BY `users`.`id` ASC LIMIT 1
D, [2021-01-18T11:44:27.364999 #7] DEBUG -- :   Document Load (1.9ms)  SELECT documents.* FROM `references` INNER JOIN `documents` ON `documents`.`id` = `references`.`document_id` WHERE `references`.`referenceable_type` = 'Entity' AND `references`.`referenceable_id` = 37687 ORDER BY `references`.`created_at` DESC LIMIT 10 OFFSET 0
D, [2021-01-18T11:44:27.365976 #7] DEBUG -- :   ↳ app/queries/recent_entity_references_query.rb:12:in `run'
I, [2021-01-18T11:44:27.367403 #7]  INFO -- : Completed 200 OK in 14ms (Views: 0.4ms | ActiveRecord: 3.8ms | Allocations: 2170)
```

It looks like `EntitiesController#references` never handled JSON, so I'm unsure why something would have been calling it.

Do you want to double-check and close if appropriate @aepyornis ?

**Comment by aepyornis** [comment]
Yes, I think that's right. It's been fixed. thanks!
